documentation/ipi-install/modules/ipi-install-upstream-troubleshooting.adoc

```diff
@@ -0,0 +1,9 @@
+[id="ipi-install-upstream-troubleshooting"]
+= Troubleshooting
+
+Troubleshooting installation has been left outside of the regular workflow and has its own link:../latest/Troubleshooting[document].
+
+[TIP]
+====
+Check link:../latest/Troubleshooting[troubleshooting] document.
+====
```

Comment:
Don't think we need the same link twice back to back. 

```
I think this could be just "Troubleshooting the installation is out of scope of the Deployment Guide. For more details on troubleshooting deployment, refer to our  link:../latest/Troubleshooting[Troubleshooting guide].
```

^ Or something to this effect.

Reply:
Changed in latest push